src/components/BccInput/BccInput.vue

```diff
@@ -9,19 +9,28 @@ type Props = {
   state?: "default" | "error" | "success";
   disabled?: boolean;
   label?: string;
+  optionalLabel?: string;
```

Comment:
We can probably default this to "optional", even though people might want to translate it then we at least provide a sensible default.